### ISSUE TITLE
.pc files for pkgconfig should be provided in addition to pcap-config

### ISSUE THREAD

**Issue by rofl0r** [the-tcpdump-group/libpcap#374]
it's a PITA to deal with proprietary pkg-config clones when cross-compiling, as most of them dont have a SYSROOT flag, so they return include paths and library paths that point to the host /usr/include... etc rather than to $SYSROOT/usr/include as they should.

working around this design bug in every single package that thought it's a good idea to reinvent pkg-config, and all other packages using this config tool, is the biggest nightmare of anyone trying to compile a distro via crosscompilers.

**Comment by tpetazzoni** [comment]
Yes, please add pkg-config support. As was expressed in the original issue, scripts like pcap-config are really not practical to use when cross-compiling. pkg-config offers a much more standard solution.

**Comment by guyharris** [comment]
And pkg-config is really not practical to use on systems that don't have pkg-config as a standard component, so, whilst I'm willing to add .pc files, I won't get rid of pcap-config, and won't get rid of its use in either tcpdump's autoconf script and CMake files or Wireshark's CMake files, although I'm willing to have them use pkg-config *if* present and *if* libpcap .pc files are found, and then try pcap-config, and then fall back on wired-in knowledge.

**Comment by guyharris** [comment]
Reopening with the new title.

**Comment by guyharris** [comment]
> Packages that use libpcap can first try with pkg-config, and if it isn't available, fallback to pcap-config.

Yes, as I said.
